vm/devices/storage/disk_nvme/nvme_driver/src/queue_pair.rs

```diff
@@ -469,3 +611,51 @@ pub(crate) fn admin_cmd(opcode: spec::AdminOpcode) -> spec::Command {
         ..FromZeroes::new_zeroed()
     }
 }
+
+#[repr(C)]
```

Comment:
Why repr(C)?

Reply:
I am sure it didn't build without it, will get back on this.

Reply:
AsBytes requires it:
error: must have a non-align #[repr(...)] attribute in order to guarantee this type's memory layout

But it looks like AsBytes is not needed anymore for the current implementation, as it was replaced with implicit conversion, will remove them both.